frontend/src/metabase/admin/permissions/components/PermissionsPageLayout/PermissionsPageLayout.tsx

```diff
@@ -10,6 +10,13 @@ import { LeaveConfirmationModal } from "metabase/components/LeaveConfirmationMod
 import Modal from "metabase/components/Modal";
 import ModalContent from "metabase/components/ModalContent";
 
+import {
+  Modal as NewModal,
+  Text,
+  Button as NewButton,
```

Comment:
Should probably follow this up by removing the depreciated `<Modal>` and `<Button>` components